ginivision-accounting-network/src/main/java/net/gini/android/vision/accounting/network/model/BoxMapper.java

```diff
@@ -0,0 +1,54 @@
+package net.gini.android.vision.accounting.network.model;
+
+import android.support.annotation.Nullable;
+
+import net.gini.android.models.Box;
+import net.gini.android.vision.network.model.GiniVisionBox;
+
+/**
+ * Created by Alpar Szotyori on 30.01.2018.
```

Comment:
Almost a year since we implemented the networking plugin...

![](https://media.giphy.com/media/l0HlVV4DMXZKhvt60/giphy.gif)